UnitsNet.Tests/BaseDimensionsTests.cs

```diff
@@ -28,13 +28,13 @@ public void ConstructorImplementedCorrectly()
         [InlineData(0, 0, 0, 0, 1, 0, 0)]
         [InlineData(0, 0, 0, 0, 0, 1, 0)]
         [InlineData(0, 0, 0, 0, 0, 0, 1)]
-        public void IsBaseImplementedSuccessfully(int length, int mass, int time, int current, int temperature, int amount, int luminousIntensity)
+        public void IsBaseQuantityImplementedProperly(int length, int mass, int time, int current, int temperature, int amount, int luminousIntensity)
         {
             var baseDimensions = new BaseDimensions(length, mass, time, current, temperature, amount, luminousIntensity);
             var derivedDimensions = new BaseDimensions(length * 2, mass * 2, time * 2, current * 2, temperature * 2, amount * 2, luminousIntensity * 2);
 
-            Assert.True(baseDimensions.IsBase());
-            Assert.False(derivedDimensions.IsBase());
+            Assert.True(baseDimensions.IsBaseQuantity());
```

Comment:
Aw gad I don't know why I keep getting confused about what are units and what are quantities with these BaseUnits thing. I keep thinking about "SI base units" I guess.